app/src/main/res/layout/activity_main.xml

```diff
@@ -55,29 +54,30 @@
 
     <EditText
         android:id="@+id/editText"
-        android:layout_width="420dp"
-        android:layout_height="50dp"
-        android:layout_marginLeft="10dp"
-        android:layout_marginRight="10dp"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/ht1"
+        android:layout_marginLeft="@dimen/marginLeft"
+        android:layout_marginRight="@dimen/marginRight"
         android:ems="10"
         android:hint="Enter your Phone Number"
         android:textColorHint="@color/black"
         android:inputType="phone"
-        android:padding="10dp"
+        android:padding="@dimen/padding"
         android:textColor="@color/black"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
 
         app:layout_constraintHorizontal_bias="0.251"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView3"
+        app:layout_constraintTop_toBottomOf="@+id/tvLogin"
         app:layout_constraintVertical_bias="0.053" />
 
     <TextView
-        android:id="@+id/textView4"
-        android:layout_width="100dp"
-        android:layout_height="20dp"
+        android:id="@+id/tvHelp"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/ht1"
```

Comment:
@raghav32 Please use wrap_content if view is to take as much space as of the text